src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -24,11 +24,18 @@
 #define _OPCODES_H
```

Comment:
Please not Opcodes.cpp changes are too many to be displayed, check https://github.com/jackpoz/TrinityCore/blob/509fd2d2e4bbfeb22b720224f2c192c44b0666e0/src/server/game/Server/Protocol/Opcodes.cpp